dataspaces.conf

```diff
@@ -2,10 +2,10 @@
 ## Config file for DataSpaces
 
 ndim = 2 
-dims = 600,600
+dims = 1600,1500
```

Comment:
@kshitij-v-mehta Thanks for catching these. They were artifacts from some earlier testing that I had left in. The dims values are the default global dimensions of the space. These must be larger than the dimensions being written by heat_transfer, but if they are too large the space will not be partitioned efficiently in the case of multiple servers. Optimal values powers of two (and the same for all dimensions), so I set them to 256,256 for the release.